cypress/integration/mocked/okta_send_reset_password.cy.ts

```diff
@@ -1,14 +1,172 @@
+    id: 'test',
+    status: 'ACTIVE',
+    profile: {
+      login: 'mrtest@theguardian.com',
+      email: 'mrtest@theguardian.com',
+    },
+    credentials: {
+      provider: {
+        type: 'OKTA',
+        name: 'OKTA',
+      },
+    },
+  };
+
+  const mockUserStaged: UserResponse = {
+    id: 'test',
+    status: 'STAGED',
+    profile: {
+      login: 'mrtest@theguardian.com',
+      email: 'mrtest@theguardian.com',
+    },
+    credentials: {
+      provider: {
+        type: 'OKTA',
+        name: 'OKTA',
+      },
+    },
+  };
+
+  const mockUserProvisioned: UserResponse = {
+    id: 'test',
+    status: 'PROVISIONED',
+    profile: {
+      login: 'mrtest@theguardian.com',
+      email: 'mrtest@theguardian.com',
+    },
+    credentials: {
+      provider: {
+        type: 'OKTA',
+        name: 'OKTA',
+      },
+    },
+  };
+
+  const mockUserRecovery: UserResponse = {
+    id: 'test',
+    status: 'RECOVERY',
+    profile: {
+      login: 'mrtest@theguardian.com',
+      email: 'mrtest@theguardian.com',
+    },
+    credentials: {
+      provider: {
+        type: 'OKTA',
+        name: 'OKTA',
+      },
+    },
+  };
+
+  const mockUserPasswordExpired: UserResponse = {
+    id: 'test',
+    status: 'PASSWORD_EXPIRED',
+    profile: {
+      login: 'mrtest@theguardian.com',
+      email: 'mrtest@theguardian.com',
+    },
+    credentials: {
+      provider: {
+        type: 'OKTA',
+        name: 'OKTA',
+      },
+    },
+  };
+
   beforeEach(() => {
     cy.mockPurge();
   });
 
-  context('send reset password email', () => {
+  context('send reset password email for ACTIVE user', () => {
+    it('shows email sent page when successful', () => {
+      cy.visit('/reset-password?useOkta=true');
+      cy.get('input[name="email"]').type(email);
+      cy.mockNext(200, mockUserActiveWithPassword);
+      cy.mockNext(200, {
+        status: 'RECOVERY_CHALLENGE',
+        factorResult: 'WAITING',
+        factorType: 'EMAIL',
+        recoveryType: 'PASSWORD',
+      });
+      cy.get('button[type="submit"]').click();
+      cy.contains('Check your email inbox');
+    });
+  });
+
+  context('send reset password email for ACTIVE user without password', () => {
     it('shows email sent page when successful', () => {
       cy.visit('/reset-password?useOkta=true');
       cy.get('input[name="email"]').type(email);
+      cy.mockNext(200, mockUserActiveWithoutPassword);
+      cy.mockNext(403, {
+        errorCode: 'E0000006',
+        errorSummary:
+          'You do not have permission to perform the requested action',
+        errorLink: 'E0000006',
+        errorId: 'errorId',
+        errorCauses: [],
+      });
+      cy.mockNext(200, {
+        resetPasswordUrl: `https://${Cypress.env(
+          'BASE_URI',
+        )}/reset_password/token_token_token_to`,
+      });
+      cy.mockNext(200, {
+        stateToken: 'stateToken',
+        expiresAt: new Date(Date.now() + 1800000 /* 30min */),
```

Comment:
should these be 60 min now we have the new timeout?

Reply:
For mocked tests it doesn't actually matter what the expiry time is.

In this case it won't be 60 mins, or 30 mins. It's actually 5 mins 30 seconds as we discovered during the testing, which is what we're fixing in https://github.com/guardian/gateway/pull/1856